embrace-android-sdk/src/main/java/io/embrace/android/embracesdk/injection/DataCaptureServiceModule.kt

```diff
@@ -95,19 +96,23 @@ internal class DataCaptureServiceModuleImpl @JvmOverloads constructor(
     }
 
     override val componentCallbackService: ComponentCallbackService by singleton {
-        ComponentCallbackService(coreModule.application, memoryService)
+        Systrace.traceSynchronous("component-callback-service-init") {
+            ComponentCallbackService(coreModule.application, memoryService)
+        }
     }
 
     override val powerSaveModeService: PowerSaveModeService by singleton {
-        if (configService.autoDataCaptureBehavior.isPowerSaveModeServiceEnabled()) {
-            EmbracePowerSaveModeService(
-                coreModule.context,
-                backgroundWorker,
-                initModule.clock,
-                systemServiceModule.powerManager
-            )
-        } else {
-            NoOpPowerSaveModeService()
+        Systrace.traceSynchronous("power-service-init") {
```

Comment:
Are these just being added as they're the biggest culprits? I was curious why these services are traced but others aren't

Reply:
Basically. I added a bunch of them and most aren't bad. I have another PR that needs more testing that starts to improve on some of these, but I'm not comfortable with getting that in so close to a release